tests/unit/test_configuration.py

```diff
@@ -662,3 +663,96 @@ def test_bad_duplicate_with_guardian_config(key: str) -> None:
                 },
             },
         )
+
+
+@pytest.mark.parametrize(
+    "noise_type, column, parameter, expected_value",
+    [
+        ("do_not_respond", None, None, 0.0145),
+        ("do_not_respond", None, "row_probability", 0.0145),
+        ("duplicate_with_guardian", None, "row_probability_in_households_under_18", 0.02),
+        ("leave_blank", "first_name", None, 0.01),
+        ("make_phonetic_errors", "first_name", "cell_probability", 0.01),
+        ("make_phonetic_errors", "first_name", "token_probability", 0.1),
+    ],
+)
+def test_working_general_noise_config_method(
+    noise_type: str, column: Optional[str], parameter: Optional[str], expected_value: float
+) -> None:
+    config = get_configuration()
```

Comment:
Would it be worth turning these two lines into a fixture? They get called in all these tests

Reply:
Seems like a good idea